src/signals/incident-management/components/FilterForm/__tests__/mockResponse.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
we already have an `src/utils/__tests__/fixtures/user.json` mock, could you use that one please?